src/Avalonia.Base/Media/BoxShadows.cs

```diff
@@ -62,7 +62,7 @@ public override string ToString()
         }
 
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public struct BoxShadowsEnumerator
+        public record struct BoxShadowsEnumerator
```

Comment:
I don't think there is any point on declaring enumerators as records. Warning should be suppressed here and other enumerators IMO. They are not compared in any way.